css/index.css

```
@@ -167,3 +263,85 @@ footer p {
 .fab:hover {
   color: #ef8376;
 }
+.container {
+  margin: 0 auto;
+  max-width: 100%;
+  width: 1400px;
+}
```

```suggestion
/// put styles here
}
```
Put all the code from line 271 to line 312 here (inside `.container.about` { }). That will make those styles more specific so they don't override my styles.